content/meetup.html

```diff
@@ -76,8 +76,8 @@
                                    target="_blank">Contributing to CarbonData</a></li>
                             </li>
                             <li>
-                                <a href="https://cwiki.apache.org/confluence/display/CARBONDATA/Committers"
-                                   target="_blank">Project Committers</a></li>
+                                <a href="https://cwiki.apache.org/confluence/display/CARBONDATA/PPMC+and+Committers+member+list"

```

Comment:
meetup.html, why need to add PPMC+Committers info ?
As i know ,the PPMC+Commtters link of Cwiki has been added to index.html, as below : 
 <li>
                                <a href="https://cwiki.apache.org/confluence/display/CARBONDATA/PPMC+and+Committers+member+list"
                                   target="_blank">Project PPMC and Committers</a></li>